Error when trying to launch a project for local development

As per this issue, which was closed despite being raised against v5: https://github.com/swagger-api/swagger-editor/issues/5302 

All screenshots are applicable to my issue

Q&A (please complete the following information)

    OS: windows 11
    Browser: Firefox
    Version: 
    Method of installation: npm
    Swagger-Editor version: 5.0.3

Content & configuration

I am trying to run swagger-editor locally
I followed the startup instructions:

 $ git submodule init
 $ git submodule update
 $ npm i
 $ npm start

and I got an error:

ERROR in ./src/plugins/editor-monaco/components/MonacoEditor/MonacoEditor.jsx 8:0-40
Module not found: Error: You attempted to import /node_modules/monaco-editor which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.

--- a/config/webpack.config.js
+++ b/config/webpack.config.js
@@ -205,8 +205,10 @@ export default (webpackEnv) => {
         // Use path.resolve to ensure cross-platform compatibility (fixes Windows path issues)
         'monaco-editor': path.resolve(paths.appNodeModules, 'monaco-editor'),
         // This alias makes sure we're avoiding a runtime error related to this package
-        '@stoplight/ordered-object-literal$':
-          path.resolve(paths.appNodeModules, '@stoplight/ordered-object-literal/src/index.mjs'),
+        '@stoplight/ordered-object-literal$': path.resolve(
+          paths.appNodeModules,
+          '@stoplight/ordered-object-literal/src/index.mjs'
+        ),
         src: paths.appSrc,
       },
       plugins: [

